zipkin-api.yaml

```diff
@@ -248,17 +248,30 @@ definitions:
     properties:
       traceId:
         type: string
+        description: 'Randomly generated globally unique transaction/context id'
       name:
         type: string
+        description: 'RPC name'
+      parentId:
+        type: string
+        maxLength: 16
+        minLength: 16
+        description: 'Parent span id (16 digit hex value)'
       id:
         type: string
+        maxLength: 16
+        minLength: 16
+        description: 'Id of current span (16 digit hex value)'
       timestamp:
         type: integer
         format: int64
+        description: 'Start time of span as UNIX timestamp in **milliseconds**'
       duration:
         type: integer
+        description: 'Duration of span in **milliseconds**'
       debug:
         type: boolean
+        description: 'Indicator whether this trace was sampled from standard traffic or forced.  Usually coupled to the X-B3-Flags header'
```

Comment:
Just so I'm not misunderstanding anything.  The header X-B3-Flags overrides the sampling policy on the service being instrumented (e.g. for debugging purposes to force a trace).  This is then recorded against the span for posterity.  There's no sampling policy per say on the zipkin service API itself correct?